taskcluster/ci/build/kind.yml

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - ffios_taskgraph.transforms.bitrise:transforms
+    - ffios_taskgraph.transforms.secrets:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+
+jobs:
+    screenshots:
+        bitrise-workflow: jlorenzo_L10nBuild
```

Comment:
I created a different workflow for en-US. This one actually compiles the task and exposes the results. This way, `generate-screenshots` tasks can reuse a compiled build.

I plan to rename this workflow once I enable all locales.